tests/testthat/test-validate_submission.R

```diff
@@ -327,20 +327,6 @@ test_that("Ignoring derived_task_ids in validate_submission works", {
       )
     )
   )
-  # Results of validation the same
-  expect_equal(
-    validate_submission(
-      hub_path = system.file("testhubs/samples", package = "hubValidations"),
-      file_path = "flu-base/2022-10-22-flu-base.csv",
-      skip_submit_window_check = TRUE,
-      derived_task_ids = "target_end_date"
-    ),
-    validate_submission(
-      hub_path = system.file("testhubs/samples", package = "hubValidations"),
-      file_path = "flu-base/2022-10-22-flu-base.csv",
-      skip_submit_window_check = TRUE
-    )
-  )
```

Comment:
Removed this because of the introduction of the new check, now when `derived_task_ids` are NULL the check is skipped and a message returned whereas if `derived_task_ids` are included, the result of the check is retuned. Hence these two situations do not return the same result anymore